migtests/tests/oracle/ssl/env.sh

```diff
@@ -0,0 +1,5 @@
+export SOURCE_DB_TYPE="oracle"
```

Comment:
We can test for ssl using some of the existing tests to run with ssl connections instead of creation some new extra tests, if possible.

Reply:
Thought about this, but i wanted to test multiple combinations - for example, with tnsadmin != oraclehome/network/admin, with a different wallet location, etc, and so I thought it'd be better to just have a separate test that was as simple as possible.

Reply:
ok, makes sense